readthedocs/builds/models.py

```diff
@@ -299,6 +299,17 @@ def last_build(self):
     def latest_build(self):
         return self.builds.order_by("-date").first()
 
+    @property
+    def latest_successful_build(self):
```

Comment:
Should we try to cache this somehow? I know we've been hitting issues with calling stuff like this repeatedly... Maybe not important in this PR, but I'd love to have a general solution for this.